demos/functions/views/demo/workbox-background-sync.hbs

```diff
@@ -0,0 +1,24 @@
+<section>
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-8 mx-auto">
+        <ol>
+          <li>Open DevTools</li>
+          <li>Go to the Console</li>
+          <li>Click this button:<br /><button class="register-event btn btn-outline-primary">Fetch Request for BG Sync</button></li>
```

Comment:
we need to make network offline before this step.

Reply:
does that work with devtools?

Secondly it still shows that the request being replayed with the log message (although I will say I'm sad there isn't more info - like a summary of requests that succeeded, failed etc).

Reply:
Doesn't it plays immediately, without it?